### PR TITLE
First commit Chinese version

### PR DESCRIPTION
**Pull request by nziyouren**
translated kusama.network/zh-CN/logos.html

### CODE REVIEW

File: zh-CN/logos.html [hunk 1]
```diff
@@ -120,7 +120,7 @@
                             </li>
                         </ul>
                     </div>
-                    <a href="http://claim.kusama.network/" target="_blank" class="btn btn-outline-primary ml-lg-3">Claim KSM</a>
+                    <a href="http://claim.kusama.network/" target="_blank" class="btn btn-outline-primary ml-lg-3">兑换 KSM</a>
```
Comment (xlc) [inline zh-CN/logos.html]:
```suggestion
                    <a href="http://claim.kusama.network/" target="_blank" class="btn btn-outline-primary ml-lg-3">领取 KSM</a>
```

File: zh-CN/logos.html [hunk 2]
```diff
@@ -306,54 +306,54 @@ <h2>Usage</h2>
...
                 </div>
                 <div class="col-12 col-sm-12 col-lg-4 mt-3 mt-lg-0">
-                    <h5>Subscribe</h5>
-                    <p>Subscribe to the newsletter to hear about Kusama updates and events.</p>
-                    <button class="btn btn-primary btn-block mb-2" data-toggle="modal" data-target="#subscribe-modal">Subscribe</button>
-                    <small class="text-muted form-text">To see how we use your information please see our <a href="privacy.html">privacy policy</a>.</small>
+                    <h5>订阅</h5>
+                    <p>订阅新闻邮件获取 Kusama 最新消息和进展</p>
+                    <button class="btn btn-primary btn-block mb-2" data-toggle="modal" data-target="#subscribe-modal">订阅</button>
+                    <small class="text-muted form-text">查看我们会怎样使用您的信息请查看我们的<a href="privacy.html">隐私政策</a>.</small>
```
Comment (xlc) [inline zh-CN/logos.html]:
```suggestion
                    <small class="text-muted form-text">想了解我们如何使用您的信息，请参阅我们的<a href="privacy.html">隐私政策</a>.</small>
```

File: zh-CN/logos.html [hunk 3]
```diff
@@ -306,54 +306,54 @@ <h2>Usage</h2>
...
+                        Kusama 是一个实验性的供社区研究和开发的网络
                     </p>
                 </div>
                 <div class="col-6 col-sm-3 col-lg-2 mt-3 mt-lg-0 pl-2">
-                    <h5>Info</h5>
+                    <h5>信息</h5>
                     <ul class="nav flex-column">
                         <li class="nav-item">
-                            <a href="http://claim.kusama.network/" target="_blank" class="nav-link">Claim KSM</a>
+                            <a href="http://claim.kusama.network/" target="_blank" class="nav-link">兑换 KSM</a>
```
Comment (xlc) [inline zh-CN/logos.html]:
```suggestion
                            <a href="http://claim.kusama.network/" target="_blank" class="nav-link">领取 KSM</a>
```

File: zh-CN/logos.html [hunk 4]
```diff
@@ -306,54 +306,54 @@ <h2>Usage</h2>
...
-                            <a href="teams.html" class="nav-link">Teams</a>
+                            <a href="teams.html" class="nav-link">团队</a>
                         </li>
                         <li class="nav-item">
-                            <a href="https://guide.kusama.network/en/latest/break/bug-bounty/" class="nav-link">Bug bounty</a>
+                            <a href="https://guide.kusama.network/en/latest/break/bug-bounty/" class="nav-link">Bug 赏金计划</a>
                         </li>
                         <li class="nav-item">
-                            <a href="https://guide.kusama.network/en/latest/code-of-conduct/" class="nav-link">Code of conduct</a>
+                            <a href="https://guide.kusama.network/en/latest/code-of-conduct/" class="nav-link">代码审计</a>
```
Comment (xlc) [inline zh-CN/logos.html]:
```suggestion
                            <a href="https://guide.kusama.network/en/latest/code-of-conduct/" class="nav-link">行为准则</a>
```

**Comment by nziyouren** [comment]
@ansonla3 @xlc I've resolved conflicts, pls help merge

**Pull request merged**